rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -3227,6 +3227,7 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
                 !it.key.startsWith( 'nodeExclude')
             }
         }else{
+            deleteExistingOptions(scheduledExecution)
```

Comment:
Try to add a javadoc to deleteExistingOptions() method explaining its purpose (I know the method name says everything but extra information is not bad and also now a javadoc is generated in every release)